src/test/java/com/google/cloud/tools/maven/run/RunnerTest.java

```diff
@@ -97,13 +95,9 @@ private void setUpAppEngineWebXml() throws IOException {
   }
 
   @Test
-  @Parameters({"1,V1", "2-alpha,V2ALPHA"})
-  public void testRun(String version, SupportedDevServerVersion mockVersion)
-      throws MojoExecutionException, IOException, AppEngineException {
+  public void testRun() throws MojoExecutionException, IOException, AppEngineException {
```

Comment:
Probably you can remove `@RunWith(JUnitParamsRunner.class)`.

Reply:
Switched to Mockito Junit Runner